lib/car.rb

```diff
@@ -0,0 +1,31 @@
+class Car
+  attr_reader :type,
+              :monthly_payment,
+              :loan_length,
+              :color
+
+  def initialize(type, monthly_payment, loan_length)
+    @type = type
+    @monthly_payment = monthly_payment
+    @loan_length = loan_length
+    @color = nil
+  end
+
+  def make
```

Comment:
Nice job using the `.split` method here. Another way you could write this to shorten it up is
```
@type.split.first
```